My Nintendo/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "author": {
+    "name": "ACertainCoder",
+    "id": "320507572455079949"
+  },
+  "service": "My Nintendo",
+  "description": {
+    "en": "Thanks to My Nintendo makes playing software and interacting with Nintendo even more fun. You can earn points and get rewards such as discounts and additional content for games.",
+    "de": "Dank My Nintendo macht das Spielen von Software und die Interaktion mit Nintendo noch mehr Spaß. Du kannst Punkte sammeln und Belohnungen wie Rabatte und Zusatzinhalte für Spiele erhalten."
+  },
+  "url": ["my.nintendo.com"],

```

Comment:
url shouldnt be in an array if its only 1

Reply:
Done 👍